### DISCUSSION TITLE
what is your build step and deployment step ?

### DISCUSSION THREAD

**Discussion by DevjayD** [dr5hn/countries-states-cities-database#340] [Q&A]
what is your build step and deployment step ?

**Comment by dr5hn** [comment]
@DevjayD Of what?

I didn't get it, Can you please explain?

**Comment by DevjayD** [comment]
Hii,

can you please clear me what exactly you  want to discuss..??

regards,
Jay Darji

**Comment by dr5hn** [comment]
> what are your build step and deployment step ?

Can you please explain your question? What do you mean by build and deployment step?

Are you using NPM package?
